src/utils/factories/user-factory.js

```diff
@@ -4,9 +4,16 @@ const UserFactory = {
   generate: () => {
     const firstName = faker.name.firstName()
     const lastName = faker.name.lastName()
+    const username = faker.internet.userName(firstName, lastName)
     return {
-      fullName: `${firstName} ${lastName}`,
-      email: faker.internet.email(firstName, lastName, faker.internet.domainName()),
+      firstName: `${firstName}`,
+      lastName: `${lastName}`,
+      username,
```

Comment:
No value for `username`

Reply:
it is using "Shorthand property names" https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer

Reply:
I see. It should be consistent tho as the others are not using it. Eg:
```javascript
firstName: `${firstName}`,
```

Also, you can drop the backticks completely for that one since no concatenation is happening

Reply:
updated